CitiesUIService/src/test/java/com/example/ApiDocumentation.java

```diff
@@ -54,4 +56,13 @@ public void indexExample() throws Exception {
         ;
     }
 
+    @Test
+    public void convertSwaggerToAsciiDoc() throws Exception {
```

Comment:
please name and structure test according to ArrangeActAssert approach, look it up.